cdc/sink/dmlsink/txn/worker.go

```diff
@@ -122,27 +117,38 @@ func (w *worker) runLoop() error {
 				zap.Int("workerID", w.ID))
 			return nil
 		case txn := <-w.txnCh.Out():
+			// we get the data from txnCh.out until no more data here or reach the state that can be flushed.
```

Comment:
`startToBatch` is not a good name.
There are 3 stages: waiting for a new batch, collecting a new batch and then flush it. Busy ratio is based on stage 2 and 3.
So I think we can record `startToCollectAndEmit` here, and then observe it after flushed.

Reply:
I think startToCollectAndEmit maybe not appropriate here, we just want to use startToBatching to calculate the total time spent in the worker.